firmware/controllers/limp_manager.cpp

```diff
@@ -75,6 +75,17 @@ todo AndreiKA this change breaks 22 unit tests?
 		allowSpark.clear();
 */
 	}
+	
+	// Fuel cut if launch control engaged
+	if (engine->launchController.isLaunchFuelRpmRetardCondition()) {
```

Comment:
do we need to check `isLaunchCondition` too?

Reply:
nope, looks like these functions check that internally. Sweet!

Reply:
it is already checked as part of bool LaunchControlBase::isLaunchRpmRetardCondition()